src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/PunishmentsMappingResourceIntTest.kt

```diff
@@ -34,7 +34,7 @@ import java.time.LocalDateTime
 import java.time.temporal.ChronoUnit
 
 const val DPS_PUNISHMENT_ID = "876"
-const val NOMIS_BOOKING_ID = 543422L
+const val NOMIS_BOOKING_ID = 5434220000
 const val NOMIS_SANCTION_SEQUENCE = 2
```

Comment:
force it to be Long to make testing easier below when they are read as a Map<String, *>